packages/preview/vartable/0.1.0/tabvar.typ

```diff
@@ -0,0 +1,358 @@
+#import "@preview/fletcher:0.4.5": *
+#let signe = "Sign"
+#let variation = "Variation"
+#let bo(x) = table(columns: 2cm, stroke: 0pt)[#table.cell(align: center + horizon)[#x]]
+
+// les deux prochaine fonctions son pour connaitre le nombre d’éléments à skip
+#let prochainNonVideSigne(x, i) = {
```

Comment:
I think this function and the two below are meant to be private. If you want to properly mark them as such, you can either prefix their name with a `_` (doesn't actually make it private, but it is a convention that is adopted in the Typst ecosystem) or move all your functions to another file and only re-export the public ones from the entrypoint.

Reply:
ah I didn't know this, I'll change it, thanks